docs/developer-guide/plugins.md

```diff
@@ -87,52 +86,55 @@ returns an observable of the response. It can be used to modify the request and/
 
 #### The intercept() method
 
-The intercept() method is called when an HTTP request is available for plugin to process. Note that, however:
+The `intercept` is a callback that exposes a proxied request to a HttpInterceptor (plugin). 
```

Comment:
"The `intercept` is" -> either "The `intercept` method is" or just "`intercept` is"

"a HttpInterceptor" -> "an `HttpInterceptor`"